### DISCUSSION TITLE
窗口函数 lead 默认值设置为时间字符串时正常，设置为now()时报错

### DISCUSSION THREAD

**Discussion by cdavis26424** [apache/doris#12554] [A - General / Q&A]
version 0.15.1

某表有字段
id varchar(255),
create_time datetime

使用下列sql可以正常执行
select id, create_time, lead(create_time, 1, '2022-09-06 00:00:00') over (order by create_time desc) as "prev_time" from test_tab;
下列sql报错，ERROR 1064 (HY000): errCode = 2, detailMessage = Type = DATETIME can't not convert to DATETIME
select id, create_time, lead(create_time, 1, date_sub(now(), interval 7 day)) over (order by create_time desc) as "prev_time" from test_tab;
select id, create_time, lead(create_time, 1,now()) over (order by create_time desc) as "prev_time" from test_tab;

用什么方式可以在default使用变量

**Comment by JNSimba** [comment]
I also have this problem trying on 1.1.2, let's record

**Comment by JNSimba** [comment]
this pr fix: https://github.com/apache/doris/pull/12587
